tests/nn/pipe_process/test_bugs.py

```diff
@@ -22,13 +22,13 @@
 from torch import nn
 import torch.nn.functional as F
 
-from fairscale.nn.pipe import AsyncPipe, MultiProcessPipe
+from fairscale.nn.pipe import AsyncPipe
 from fairscale.utils.testing import get_worker_map, torch_spawn
 
 
 @torch_spawn([2])
 @pytest.mark.skipif(not torch.cuda.is_available(), reason="cuda required")
-@pytest.mark.parametrize("pipe_class", [MultiProcessPipe, AsyncPipe])
+@pytest.mark.parametrize("pipe_class", [AsyncPipe])
```

Comment:
nit: you don't need parametrize anymore with this one option. maybe you are planning to add another option soon in which case it makes sense.